code/06-building-decays/build_decays.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# =============================================================================
+# @file   build_decays.py
+# @author Albert Puig (albert.puig@cern.ch)
+# @date   29.05.2015
+# =============================================================================
```

Comment:
I have `git blame` to find out who to apply my baseball bat to if need be (as in I don't need the name in the top of the file) ;)